rcaracaus/pledge-vegan-node
# Fixed missing title on contact us page.

Steps to test:

1. Go to localhost:3000/contact-us
2. Check that there is a h1 on the top of the page "Contact Us"
